config/externalname.go

```diff
@@ -602,6 +602,7 @@ var TerraformPluginSDKExternalNameConfigs = map[string]config.ExternalName{
 	"azurerm_private_dns_zone_virtual_network_link":     config.TemplatedStringAsIdentifier("name", "/subscriptions/{{ .setup.configuration.subscription_id }}/resourceGroups/{{ .parameters.resource_group_name }}/providers/Microsoft.Network/privateDnsZones/{{ .parameters.private_dns_zone_name }}/virtualNetworkLinks/{{ .external_name }}"),
 	"azurerm_private_link_service":                      config.TemplatedStringAsIdentifier("name", "/subscriptions/{{ .setup.configuration.subscription_id }}/resourceGroups/{{ .parameters.resource_group_name }}/providers/Microsoft.Network/privateLinkServices/{{ .external_name }}"),
 	"azurerm_private_endpoint":                          config.TemplatedStringAsIdentifier("name", "/subscriptions/{{ .setup.configuration.subscription_id }}/resourceGroups/{{ .parameters.resource_group_name }}/providers/Microsoft.Network/privateEndpoints/{{ .external_name }}"),
+	"azurerm_private_dns_resolver_inbound_endpoint":     config.TemplatedStringAsIdentifier("name", "/subscriptions/{{ .setup.configuration.subscription_id }}/resourceGroups/{{ .parameters.resource_group_name }}/providers/Microsoft.Network/dnsResolvers/{{ .parameters.azurerm_private_dns_resolver }}/inboundEndpoints/{{ .external_name }}"),
```

Comment:
In the Terraform document, in the import section, I see the resource id as `/subscriptions/00000000-0000-0000-0000-00000000000/resourceGroups/resourceGroup1/providers/Microsoft.Network/dnsResolvers/dnsResolver1/inboundEndpoints/inboundEndpoint1`.  

This part `/subscriptions/00000000-0000-0000-0000-0000000000/resourceGroups/resourceGroup1/providers/Microsoft.Network/dnsResolvers/dnsResolver1` seems to be equal to the id of [azurerm_private_dns_resolver](https://github.com/crossplane-contrib/provider-upjet-azure/blob/970ba77a7bc901362ba0f60eb7afe2e7a86ebff1/config/externalname.go#L1840) resource and the resource we want to add has `private_dns_resolver_id` in its parameters. So can you try the configuration below?

```suggestion
	"azurerm_private_dns_resolver_inbound_endpoint":     config.TemplatedStringAsIdentifier("name", "{{ .parameters.private_dns_resolver_id}}/inboundEndpoints/{{ .external_name }}"),
```